src/v1/components/surfaces/AppBar/AppBar.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { userEvent, within } from "@storybook/test";
+
+import AppBar from "./AppBar";
+import { Text, UserProfile, UserStatus } from "../../data-display";
+
+const meta: Meta<typeof AppBar> = {
+  title: "Surfaces/AppBar",
+  component: AppBar,
+  tags: ["autodocs"],
+  parameters: {
+    layout: "fullscreen",
+  },
+};
+export default meta;
+
+type Story = StoryObj<typeof AppBar>;
+
+export const Example: Story = {};
+
+export const WithAppName: Story = {
+  args: {
+    appName: "Design System",
+  },
+};
+
+export const WithUserProfile: Story = {
+  args: {
+    ...WithAppName.args,
+    userProfile: (
+      <UserProfile fullName="Han Solo">
+        <UserStatus fullName="Han Solo">
+          <Text variant="subtitle1">Roles</Text>
+          <Text>Smuggler</Text>
+          <Text>Scoundrel</Text>
+          <Text>Hero</Text>
+        </UserStatus>
+      </UserProfile>
+    ),
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+
+    await userEvent.click(canvas.getByRole("button", { name: "user-profile" }));
+  },
+};
```

Comment:
For now I don't care where these components are placed

More important to identify the guidance for placing: [Any thoughts here?](https://telicent.atlassian.net/wiki/spaces/FTS/pages/21692670/Design+system+guidance#Deciding-on-where-components-go) (apart from being messy heh)

Reply:
Are you proposing having a component library and a design system?

Reply:
More "wondering" if we need it

Reply:
@ACoolmanTelicent will have further discussions about this